pkg/graph/schema/types/operational_solution_and_model_plan.graphql

```diff
@@ -0,0 +1,22 @@
+enum StatusPlannedActiveOrEnded {
+  PLANNED
+  ACTIVE
+  ENDED
+}
+
+extend type ModelPlan {
+  statusPlannedActiveOrEnded: StatusPlannedActiveOrEnded!
+}
+
+type ModelPlanAndOperationalSolution {
+  # TODO -- we can uncomment these if we want?

```

Comment:
Can you either uncomment this / remove the comment?

Sorry -- I definitely threw this in last minute before handing the branch over!